### PR TITLE
MF 1413 - Use per-service URL in SDK

### LINKED ISSUE #1413: Use per-service URL in SDK
**ENHANCEMENT**
1. Describe the enhancement you are requesting.
Currently SDK uses `BaseURL` field, as it was supposed that will be used by a client that will be hitting Mainflux in the cloud, behind the proxy and with the single URL.

However, this prevents SDK being used by microservices internally, in the private Docker network, as then need to communicate in the service mesh fashion, so each microservice must be capable to address any other - we need SDK not to keep only `BaseURL`, but URLs of each microservice.

2. Indicate the importance of this enhancement to you (must-have, should-have, nice-to-have).
Must have

### PR DESCRIPTION
**Pull request by drasko**
Signed-off-by: Drasko DRASKOVIC <drasko.draskovic@gmail.com>

### What does this do?
Fixes SDK to use per-service URL instead of `BaseURL`.

### Which issue(s) does this PR fix/relate to?
Resolves #1413

### CODE REVIEW

File: cmd/cli/main.go [hunk 1]
```diff
@@ -11,21 +11,19 @@ import (
...
-		HTTPAdapterPrefix: "http",
-		BootstrapPrefix:   "things",
-		MsgContentType:    sdk.ContentType(msgContentType),
-		TLSVerification:   false,
+		AuthURL:         defURL,
+		ThingsURL:       defURL,
+		UsersURL:        defURL,
+		ReaderURL:       defURL,
+		BootstrapURL:    defURL,
+		CertsURL:        defURL,
```
Comment (manuio) [inline cmd/cli/main.go:24]: Are you sure here?

File: bootstrap/api/endpoint_test.go [hunk 2]
```diff
@@ -160,7 +160,7 @@ func dec(in []byte) ([]byte, error) {
 func newService(auth mainflux.AuthServiceClient, url string) bootstrap.Service {
 	things := mocks.NewConfigsRepository()
 	config := mfsdk.Config{
-		BaseURL: url,
+		BootstrapURL: url,
```
Comment (mteodor) [inline bootstrap/api/endpoint_test.go:163]: you dont have ThingsURL here

File: bootstrap/api/endpoint_test.go [hunk 3]
```diff
@@ -238,7 +238,7 @@ func TestAdd(t *testing.T) {
 			auth:        validToken,
 			contentType: contentType,
 			status:      http.StatusCreated,
-			location:    "/things/configs/1",
+			location:    "/configs/1",
```
Comment (mteodor) [inline bootstrap/api/endpoint_test.go:241]: have you changed `responses.go`

File: bootstrap/api/endpoint_test.go (hunk 2 above)
Comment (drasko) [reply bootstrap/api/endpoint_test.go:163]: Why do we need it and where is defined? In an ENV var?

**Pull request closed without merging**